txp/Pos/Txp/Toil/Logic.hs

```diff
@@ -130,8 +168,10 @@ verifyAndApplyTx
     :: (MonadUtxo m, MonadToilEnv m, MonadError ToilVerFailure m)
     => Bool -> (TxId, TxAux) -> m TxUndo
 verifyAndApplyTx verifyVersions tx@(_, txAux) = do
-    verifyToilEnv txAux
-    Utxo.verifyTxUtxo ctx txAux <* applyTxToUtxo' tx
+    applyTxToUtxo' tx
```

Comment:
Accidentally reordered actions here, so that `applyTxToUtxo'` is called before verification. This is fixed in a later commit.